cocos/2d/CCLabelTextFormatter.cpp

```diff
@@ -80,14 +80,19 @@ int Label::getFirstCharLen(const std::u16string& utf16Text, int startIndex, int
 int Label::getFirstWordLen(const std::u16string& utf16Text, int startIndex, int textLen)
 {
     auto character = utf16Text[startIndex];
-    if (StringUtils::isCJKUnicode(character) || StringUtils::isUnicodeSpace(character) || character == (char16_t)TextFormatter::NewLine)
+    if (StringUtils::isCJKUnicode(character)
+        || StringUtils::isUnicodeSpace(character)
+        || character == (char16_t)TextFormatter::NewLine)
     {
         return 1;
     }
 
     int len = 1;
     FontLetterDefinition letterDef;
-    auto nextLetterX = 0;
+    if(_fontAtlas->getLetterDefinitionForChar(character, letterDef) == false) {
```

Comment:
Uses `if (!balabala)` instead of `if (balabala == false)` for boolean type compare.